debug/handler/debug.go

```diff
@@ -39,3 +45,66 @@ func (d *Debug) Stats(ctx context.Context, req *proto.StatsRequest, rsp *proto.S
 	rsp.Threads = uint64(runtime.NumGoroutine())
 	return nil
 }
+
+func (d *Debug) Logs(ctx context.Context, req *proto.LogRequest, stream proto.Debug_LogsStream) error {
+	var options []log.ReadOption
+
+	since := time.Unix(0, req.Since)
```

Comment:
I think it's supposed to be time.Unix(req.Since, 0) right?

Reply:
Depends on the resolution (https://play.golang.org/p/aVkhnRgurCS); I think we use `nanoseconds` in RPC requests, no? I feel like the `router` does...

Reply:
The proto says we use a unix seconds timestamp

Reply:
Damn. Good catch. Will change this tomorrow. (and the below)